### DISCUSSION TITLE
How to join DMV tables?

### DISCUSSION THREAD

**Discussion by iww9** [DaxStudio/DaxStudio#958] [Q&A] [answered]
Hello,

I'm attempting to get the calculated column names, their definitions, and the table name.

I want to join TMSCHEMA_Tables onto TMSCHEMA_COLUMNS to get the table name.

How do I join tables in Dax Studio? not a left join operation?

![image](https://user-images.githubusercontent.com/114238031/191971975-7b0a17ca-16cf-4a38-a081-8edff5e90f47.png)

**Comment by marcosqlbi** [comment, marked as answer]
It does not depend on DAX Studio - the SQL language supported for DMVs is very limited and you can only query one table at a time without joins. That's the reason why we use the VertiPaq Analyzer library in DAX Studio: https://github.com/sql-bi/VertiPaq-Analyzer

**Reply by iww9** [reply]
Thanks
